TCAT/Cells/BusStopCell.swift

```diff
@@ -44,7 +44,7 @@ class BusStopCell: UITableViewCell {
 
         textLabel?.frame = CGRect(x: labelXPosition, y: 0.0, width: frame.width - labelWidth, height: labelHeight)
         textLabel?.center.y = bounds.height / 2.0
-        textLabel?.font = .systemFont(ofSize: 13)
+        textLabel?.font = UIFont.style(Fonts.System.regular, size: 13)
```

Comment:
thoughts on dropping `UIFont` and doing just `.style(...)`

Reply:
Didn't know you could do that for a function! (not sure why, it makes sense)